packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [3.7.0]
+### Breaking change:
```

Comment:
If this is really a breaking change the major version needs to be updated -> 4.0.0

Reply:
It's not a breaking change per se. The functionality will use different env variables for configuration in the cli (not really a package/plugin). But if we consider that functionality-wise it is different, then it would make sense to upgrade the version.

Reply:
We either need to remove that statement "### Breaking change:" or to bump the major version